sdk/monitor/monitor-opentelemetry-exporter/swagger/README.md

```diff
@@ -8,7 +8,7 @@ From `swagger/`:
 
 ```zsh
 rm -rf ../src/generated
-autorest --typescript --v3
+autorest --typescript swagger/README.md
```

Comment:
You could add `typescript: true` to the configurations list in this file directly so you do not have to type it everytime.